### PR TITLE
13.0 mig web widget color

### PR DESCRIPTION
**Pull request by ammarofficewla**

### CODE REVIEW

File: web_widget_color/README.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+~~~~~~~~~~~~
+
+* Adil Houmadi <adil.houmadi@gmail.com>
+* Enric Tobella <etobella@creublanca.es>
+* Nicolas JEUDY (Sudokeys) <https://www.github.com/njeudy>
+* Alexandre Díaz <dev@redneboa.es>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Ernesto Tejeda
+* Ammar Offciewala <https://twitter.com/AmmarOfficewala>
```
Comment (sudhir-erpharbor) [inline web_widget_color/README.rst]: It must be your email address.

File: web_widget_color/README.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/web <https://github.com/OCA/web/tree/12.0/web_widget_color>`_ project on GitHub.
```
Comment (sudhir-erpharbor) [inline web_widget_color/README.rst]: 13.0

**Review by sudhir-erpharbor — APPROVED** [review]
LGTM

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please add to `view_base_document_layout` the widget in `primary_color` and `secondary_color` for both showcasing the widget and for improving that functionality, but making sure the output format is the expected one (I think so).

**Comment by ammarofficewla** [comment]
> Please add to `view_base_document_layout` the widget in `primary_color` and `secondary_color` for both showcasing the widget and for improving that functionality, but making sure the output format is the expected one (I think so).

Sorry, I have not understood your change request.

**Comment by pedrobaeza** [comment]
Modify that view for adding the widget in the specified fields.

**Comment by sbidoul** [comment]
Is this still necessary? There is a `color` widget in Odoo CE that seems to do the same.

**Comment by pedrobaeza** [comment]
This one is definitively not needed, so closing and amending OpenUpgrade part in https://github.com/OCA/OpenUpgrade/pull/2170

**Pull request closed without merging**